core/src/main/scala/org/apache/spark/deploy/SparkHadoopUtil.scala

```diff
@@ -123,7 +123,9 @@ class SparkHadoopUtil extends Logging {
    */
   def addCredentials(conf: JobConf): Unit = {
     val jobCreds = conf.getCredentials()
-    jobCreds.mergeAll(UserGroupInformation.getCurrentUser().getCredentials())
+    val userCreds = UserGroupInformation.getCurrentUser().getCredentials()
+    logInfo(s"Adding user credentials: ${SparkHadoopUtil.get.dumpTokens(userCreds)}")
```

Comment:
Don't use `dumpTokens` in an info message. Ok to add as debug if you want. But not really sure why you're adding this in this PR.

Reply:
ok.